src/controllers/playback/video/index.js

```diff
@@ -631,6 +632,17 @@ export default function (view) {
         }
     }
 
+    function onPromptSkip(e, mediaSegment) {
+        const player = this;
+        if (mediaSegment && player && mediaSegment.EndTicks != null
+            && mediaSegment.Type === MediaSegmentType.Outro
```

Comment:
Should we hardcode the check for `Outro` segments? What if it is something like a recording with a commercial at the end?

Reply:
Sorry, I'm not sure I understand.

Reply:
I'm wondering if we need to check the segment type at all here. An outro might not be the last segment in a file. 🤷‍♂️

Reply:
It does make sense to remove this check as it can apply to other segment types also.